upgrading/rosa-upgrading-sts.adoc

```diff
@@ -11,6 +11,8 @@ toc::[]
 
 To plan an upgrade, review the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle]. The life cycle page includes release definitions, support and upgrade requirements, installation policy information and life cycle dates.
 
+Upgrades are customer-initiated or automatically scheduled. The SRE team monitors upgrade progress and remedies any issues encountered.
```

Comment:
I wasn't sure about customer-initiated and didn't see any use in the docs for `customer-initiated` or `customer initiated`.  I do see `user-initiated` in the docs though https://docs.openshift.com/container-platform/4.11/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html#how-the-migration-process-works_migrate-from-openshift-sdn